lib/beaker-answers/versions/version20162.rb

```diff
@@ -103,7 +103,7 @@ def answer_hiera
       hocon.render
     end
 
-    def installer_configuration_string(host)
+    def installer_configuration_string(_host)
```

Comment:
Why is this changed to `_host`? And, it doesn't seem to be referenced in the `answer_hiera` method that this method wraps, so...maybe it can just be removed? The argument, I mean...

Reply:
Yeah, that's rubocop's idea; if a variable is unused, it marks with with a `_` so that you can review it and decide if it is even necessary.